boards/arm/faze/CMakeLists.txt

```diff
@@ -7,6 +7,6 @@
 set_property(GLOBAL APPEND PROPERTY extra_post_build_commands
   # Insert checksum (verified by the bootloader) into the zephyr.bin
   # and zephyr.hex images.
-  COMMAND lpc_checksum -f hex ${CMAKE_BINARY_DIR}/zephyr/zephyr.hex
-  COMMAND lpc_checksum -f bin ${CMAKE_BINARY_DIR}/zephyr/zephyr.bin
+  COMMAND lpc_checksum -f hex ${CMAKE_BINARY_DIR}/zephyr/${KERNEL_HEX_NAME}
+  COMMAND lpc_checksum -f bin ${CMAKE_BINARY_DIR}/zephyr/${KERNEL_BIN_NAME}
```

Comment:
There should have been a `find_program(LPC_CHECKSUM lpc_checksum)` so that it could have been ensured that the program is indeed available before this command tries to execute.

Reply:
@galak FYI